### PR TITLE
Updated Roles and Permissions

### PR DESCRIPTION
**Pull request by JohnRajunas**
I am making these suggestion to better reflect Roles and Permissions options that are currently available in the Microsoft Store for Business tenant, as well as the text and titles that appear in the MSfB/MSfE service currently. I am proposing to remove the "Global Administrator and Billing Administration section since the "Billing Administrator" is not assignable in the current release, and I believe could lead to confusion.  I also made some other minor wording changes that I believe might make the topic more easily consumed. I hope this fork will be considered

### CODE REVIEW

**Comment by Justinha** [comment]
@TrudyHa can you take a look?

**Comment by TrudyHa** [comment]
There are a few things here I need to verify with a member from Engineering, and he's out on vacation this week. More soon.

**Comment by mypil** [comment]
@TrudyHa - Do you have any updates on this? Thanks

**Comment by mypil** [comment]
@TrudyHa - We would like to follow up. Do you have any updates on this? Thanks

**Comment by mypil** [comment]
@TrudyHa - We would like to follow up. Do you have any updates on this? Thanks

**Comment by mypil** [comment]
@TrudyHa @lizap @Justinha @mjcaparas @jaimeo @DaniHalfin - Can anyone of you assist to check, approve and merge the content of this PR?

Thank you.

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by DaniHalfin** [comment]
Bigger issue right now is that this PR has conflicts. I wanted to resolve them, but didn't want to overwrite the contributor's original intent. 
If @JohnRajunas can resolve the conflict, @TrudyHa should review.

File: store-for-business/roles-and-permissions-microsoft-store-for-business.md [hunk 1]
```diff
@@ -14,34 +14,24 @@ ms.localizationpriority: medium
...
 
-|                                |  Global Administrator | Billing Administrator |
-| ------------------------------ | --------------------- | --------------------- |
-| Sign up for Microsoft Store for Business and Education |  X                    |
-| Modify company profile settings | X                    |                       |
-| Purchase apps                   |  X                    | X                     |
-| Distribute apps                |  X                    | X                     |
-| Purchase subscription-based software  |  X             | X                     |
- 
+IT Pros with Global Administrator permissions have full access to manage the Microsoft Store for Business or Microsoft Store for Education tenant. The user who signs up for a Microsoft Store for Business or Microsoft Store for Education tenant is granted this Global Admin role. They can do everything allowed by the other roles, and would assign permissions to other employees as a part of the tenant set-up process.
```
Comment (JohanFreelancer9) [inline store-for-business/roles-and-permissions-microsoft-store-for-business.md]:
```suggestion
Users with Global Administrator permissions have full access to manage the Microsoft Store for Business or Microsoft Store for Education tenant. Users who sign up for a Microsoft Store for Business or Microsoft Store for Education tenant are granted this Global Admin role. They can do everything allowed by the other roles, and would assign permissions to other employees as a part of the tenant set-up process.
```

File: store-for-business/roles-and-permissions-microsoft-store-for-business.md [hunk 2]
```diff
@@ -14,34 +14,24 @@ ms.localizationpriority: medium
...
 
-**Global Administrator** - IT Pros with this account have full access to Microsoft Store. They can do everything allowed in the Microsoft Store Admin role, plus they can sign up for Microsoft Store.
-
-**Billing Administrator** - IT Pros with this account have the same permissions as Microsoft Store Purchaser role.
 
 ## Billing account roles and permissions
-There are a set of roles, managed at your billing account level, that help IT admins and employees manage access to and tasks for Microsoft Store. Employees with these roles will need to use their Azure AD account to access Microsoft Store for Business.
+
+There are a set of roles, referred to as "Billing Account" roles, that are assigned to admins and employees who will use the Microsoft Store for Business or Microsoft Store for Education tenant to buy and manage Microsoft Store apps, and/or administer account details and billing information, for an organization. Employees with these roles will need to use their Azure AD account to access the tenant.
```
Comment (JohanFreelancer9) [inline store-for-business/roles-and-permissions-microsoft-store-for-business.md]:
```suggestion
There is a set of roles, referred to as "Billing Account" roles, that is assigned to admins and employees who will use the Microsoft Store for Business or Microsoft Store for Education tenant to buy and manage Microsoft Store apps, and/or administer account details and billing information for an organization. Employees with these roles will need to use their Azure AD account to access the tenant.
```

File: store-for-business/roles-and-permissions-microsoft-store-for-business.md [hunk 3]
```diff
@@ -62,16 +52,18 @@ These permissions allow people to:
     - Admins can assign apps to people, add apps to the private store, or use a management tool.
     - Purchasers can assign apps to people.
     --> 
+    
 ## Purchasing roles and permissions
-There are also a set of roles for purchasing and managing items bought. 
+There are also a set of roles, referred to as "Purchasing roles", that can be assigned to users to allow them to use the service to buy and manage apps for the organization at large, or for themselves.
```
Comment (JohanFreelancer9) [inline store-for-business/roles-and-permissions-microsoft-store-for-business.md]:
```suggestion
There is also a set of roles, referred to as "Purchasing roles", that can be assigned to users to allow them to use the service to buy and manage apps for the organization at large, or for themselves. 
```

File: store-for-business/roles-and-permissions-microsoft-store-for-business.md [hunk 4]
```diff
@@ -81,6 +73,6 @@ This table lists the roles and their permissions.
     >You need to be a Global Administrator, or have the Billing account owner role to access **Permissions**. 
     
 2.  Select **Manage**, and then select **Permissions**.
-3.  On **Roles**, or **Purchasing roles**, select **Assing roles**. 
+3.  On **Roles**, or **Purchasing roles**, select **Assign roles**. 
 4.  Enter a name, choose the role you want to assign, and select **Save**.
-    If you don't find the name you want, you might need to add people to your Azure AD directory. For more information, see [Manage user accounts](manage-users-and-groups-microsoft-store-for-business.md).
\ No newline at end of file
+    If you don't find the name you want, you might need to add people to your Azure AD directory. For more information, see [Manage user accounts](manage-users-and-groups-microsoft-store-for-business.md).
```
Comment (JohanFreelancer9) [inline store-for-business/roles-and-permissions-microsoft-store-for-business.md]:
```suggestion
    If you don't find the name you want, you might need to add the name to your Azure AD directory. For more information, see [Manage user accounts](manage-users-and-groups-microsoft-store-for-business.md).
```

**Comment by jaimeo** [comment]
@TrudyHa should review.

**Comment by DaniHalfin** [comment]
The default branch for this repository has changed. Commenting to remember to merge this to the new default, if this PR gets merged.

**Comment by e0i** [comment]
@JohnRajunas 

The default branch for this repository has changed from `master` to `public`. Please edit your pull request to change the merge target to `public` via the `Edit` button at the top right corner. Thanks.

**Comment by illfated** [comment]
Will be interesting to see if changing from `master` branch to `public` will make any change in the file conflict, or if it stays the same.

**Comment by Dansimp** [comment]
@JohnRajunas, unfortunately I'm going to need to close out this PR. We greatly appreciate the time and effort you've put into helping improve these docs. The issue is that there are some upcoming changes to the Store and Commerce platforms which will require additional changes here (and that's not quite ready yet). Given we've been trying to get answers for several months now with little progress, I'm going to need to close this out.

**Pull request closed without merging**